GSoC-2025/GSoC NIU Projects 2025 BrainGlobe:Add to BrainGlobe's data visualization tools (Kirato Yoshihara).md

```diff
@@ -0,0 +1,258 @@
+
+- **Implementation**
+
+  A bullet point list with **minimal set of deliverables**:
+
+  - A Python implementation of a napari widget that allows users to download and visualize atlas-registered data.
+  - Tests to cover any added functionality.
+  - Documentation for the new functionality.
+  - Preparation of mid-term and final reports.
+
+- **Implementation timeline**
+
+<table>
+  <thead>
+    <tr>
+      <th>Timeline</th>
+      <th>Deliverables</th>
+      <th>Hours</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><b>Community Bonding Period</b><br>Mon 5/8 - Sun 6/1</td>
+      <td>
+        <ol type="1">
+          <li>Discuss project details with mentor and understand the codebase</li>
+          <li>Study the <a href="https://elifesciences.org/articles/65751" target=_blank>paper</a> on atlas-registered data and visualization methods</li>
+          <li>Set up the development environment and resolve dependencies</li>
+          <li>Analyze existing napari plugin structure and brainrender integration</li>
+          <li>Identify public atlas data sources to be integrated</li>
+        </ol>
+      </td>
+      <td>20</td>
+    </tr>
+    <tr>
+      <td><b>Week 1</b><br>Mon 6/2 - Sun 6</br><i>(Exam period)</i></td>
+      <td>
+        <ol type="1">
+          <li>Design widget UI and create technical specifications</li>
+        </ol>
+      </td>
+      <td>5</td>
+    </tr>
+    <tr>
+      <td><b>Week 2</b><br>Mon 6/9 - Sun 6/15</td>
+      <td>
+        <ol type="1">
+          <li>Implement basic plugin widget framework</li>
+          <li>Implement basic data download and caching functionality</li>
+        </ol>
+      </td>
+      <td>25</td>
+    </tr>
+    <tr>
+      <td><b>Week 3</b><br>Mon 6/16 - Sun 6/22</td>
+      <td>
+        <ol type="1">
+          <li>Implement connection to mouse brain atlas</li>
+          <li>Implement basic brain region display functionality</li>
+          <li>Test initial features and fix bugs</li>
+        </ol>
+      </td>
+      <td>25</td>
+    </tr>
+    <tr>
+      <td><b>Week 4</b><br>Mon 6/23 - Sun 6/29</td>
+      <td>
+        <ol type="1">
+          <li>Enhance mouse brain detailed region selection and display</li>
+          <li>Implement basic visualization controls (color, transparency)</li>
+          <li>Refine based on mentor's feedback</li>
+        </ol>
+      </td>
+      <td>25</td>
+    </tr>
+    <tr>
+      <td><b>Week 5</b><br>Mon 6/30 - Sun 7/6</td>
+      <td>
+        <ol type="1">
+          <li>Add support for cell coordinate (Points) data</li>
```

Comment:
Also points come for free in `napari` so we don't need to support this (on the UI/frontend side at least!).